### PR TITLE
Add sqlc example

### PR DESCRIPTION
**Pull request by matbur**
I hope this example will save some time other developers would spend struggling with using [sqlc](https://docs.sqlc.dev/en/stable/) with encore.

### CODE REVIEW

File: sqlc-database/Makefile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+sqlc:
```
Comment (eandre) [inline sqlc-database/Makefile:1]: We don't use makefiles in the other examples, would be better to just mention in the README for this example to use `sqlc generate`

File: sqlc-database/go.mod [hunk 2]
```diff
@@ -0,0 +1,5 @@
+module encore.app
+
+go 1.18
+
+require encore.dev v1.1.0
```
Comment (eandre) [inline sqlc-database/go.mod:5]: Can you bump this to the latest Encore release (`go get encore.dev@latest`)?

File: sqlc-database/hello/store/queries/queries.sql [hunk 3]
```diff
@@ -0,0 +1,9 @@
+-- name: InsertGreeting :one
```
Comment (eandre) [inline sqlc-database/hello/store/queries/queries.sql:1]: This isn't inserting a greeting but rather it's incrementing the meeting count. How about naming it `IncrementMeetingCount`?

**Review by eandre — COMMENTED** [review]
Hey @matbur, thanks so much for doing this! Overall it looks great. Can you take a look at the comments below and then this should be good to merge?

**Comment by eandre** [comment]
Thanks @matbur, this looks great. I noticed the Encore CLA bot wasn't picking up this repository. If you don't mind signing the CLA we can get it merged as soon as that's done. Thank you!

**Comment by matbur** [comment]
@eandre, I signed CLA.

**Review by eandre — APPROVED** [review]

**Comment by eandre** [comment]
Thank you!

**Pull request merged**